### PR TITLE
[Config schema] Add hard newlines

### PR DESCRIPTION
**Pull request by gewarren**
- add hard newline with trailing backslash
- shorten descriptions to be more to-the-point

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/index.md [hunk 1]
```diff
@@ -22,79 +22,78 @@ helpviewer_keywords:
...
 
-[WIF Configuration Schema](windows-identity-foundation/index.md)
+[WIF Configuration Schema](windows-identity-foundation/index.md)\
 All elements of the Windows Identity Foundation (WIF) configuration schema.
 
 ## Related sections
 
-[Remoting Settings Schema](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/z415cf9a(v=vs.100))
+[Remoting Settings Schema](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/z415cf9a(v=vs.100))\
 Describes the elements that configure client and server applications that implement remoting.
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/index.md]: I think you missed this one, as well as some others below.

Comment (BillWagner) [reply docs/framework/configure-apps/file-schema/index.md]: These changes are correct. There is a blank line here and in the following elements

**Review by BillWagner — APPROVED** [review]
these all LGTM @gewarren 

You can :shipit: when ready.

**Comment by gewarren** [comment]
Thanks for the reviews; I'm merging now.

File: docs/framework/configure-apps/file-schema/index.md (hunk 1 above)
Comment (Youssef1313) [reply docs/framework/configure-apps/file-schema/index.md]: @BillWagner, I meant the line that begins with `Describes the`, similar lines was updated and the `Describes the` was deleted.

**Pull request merged**